### ISSUE TITLE
wrong leadcount

### ISSUE THREAD

**Issue by OlafDammann** [mautic/mautic#1489]
When I go "manage leads", I see my lists and at the vey right side the button "view XYZ leads" The count XYZ is at all of my lists one to high. funny but true, I counted them.

**Comment by dongilbert** [comment]
I just checked my local dev install, and my list says `View 6 Leads`, and when I click into it there's only 4. Very strange indeed. I'll look into it.

**Comment by dongilbert** [comment]
@OlafDammann We've found the issue, and it's that Anonymous leads (leads without a first name, email address, or other identifying factor that is also displayed on the leads page) are excluded from the leads page when viewing them. We're working on a solution.

**Comment by dongilbert** [comment]
@OlafDammann, can you test something? What happens when you click "View X Leads" from the lead list view, then on the right above the "Last Active" column of the filtered leads page, you click the "Toggle Anonymous Leads" button? Does the missing lead show up?

![screen shot 2016-03-02 at 1 44 49 pm](https://cloud.githubusercontent.com/assets/718028/13471067/fbd7335a-e07c-11e5-98ca-8c036313527d.png)

**Comment by OlafDammann** [comment]
Good morning!
Thanks for taking care! So, I did as you asked in the lists and found a User called "Anonymous" in each one. In all the small Lists that was the one difference that was left. (some others I found out in the mean time, had mainly to do with me messing up the Import....)
I imported all the Leads I have with a csv-File. This Import always created this Anonymous User, I doublechecked it. Maybe the 1st Line of the csv-File will be interpreted in some way?
Thansk a lot for helping out :)
OLAF
